mods/flowers/init.lua

```diff
@@ -111,9 +106,7 @@ minetest.register_abm({
 			if not light or light < 13 then
 				return
 			end
-			if minetest.get_node(seedling).name == "air" then
-				minetest.set_node(seedling, {name = node.name})
-			end
```

Comment:
I'm not sure if this check for air can be safely removed. For example, fences can be above dirt_with_grass and may have a high enough light value.